src/main/scala/com/ibm/couchdb/api/builders/GetManyDocsQueryBuilderAllowMissing.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2015 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ibm.couchdb.api.builders
+
+import com.ibm.couchdb._
+import com.ibm.couchdb.core.Client
+import org.http4s.Status
+import upickle.default.Aliases.{R, W}
+import upickle.default.write
+
+import scalaz.concurrent.Task
+
+case class GetManyDocsQueryBuilderAllowMissing(client: Client,
```

Comment:
What about adding two new methods (queryAllowMissing and queryIncludeDocsAllowMissing) to GetManyDocumentsQueryBuilder and reverting the changes to Documents?